<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobCompany</class>
 <widget class="QWidget" name="JobCompany">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>483</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="jobGroup">
     <property name="title">
      <string>Jobs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="createGroup">
     <property name="title">
      <string>Create Job Opportunity</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="jobTitleEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Job Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="workplaceLabel">
        <property name="text">
         <string>Workplace Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="workplaceCombo">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>On Site</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Remote</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hybrid</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="jobLocationEdit">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Job Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="jobTypeLabel">
        <property name="text">
         <string>Job Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="jobTypeCombo">
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>Full Time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Part Time</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Contract</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Temporary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Other</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Volunteer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Internship</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="createJobButton">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
